vault/expiration.go

```diff
@@ -1003,12 +1022,18 @@ func (m *ExpirationManager) updatePendingInternal(le *leaseEntry, leaseTotal tim
 		timer := time.AfterFunc(leaseTotal, func() {
```

Comment:
Can this section to the end of the function be simplified to:
```
	// Create entry if it does not exist,
	// otherwise extend the timer by the lease total
	if !ok {
		pending.timer = time.AfterFunc(leaseTotal, func() {
			m.expireID(le.LeaseID)
		})
	} else {
		pending.timer.Reset(leaseTotal)
	}

	pending.exportLeaseTimes = m.leaseTimesForExport(le)
	m.pending[le.LeaseID] = pending
```